README.md

```diff
@@ -7,15 +7,23 @@ undergraduate research course at the [University of Saskatchewan][usask].
 
 ## Building Miri
 
-I recommend that you install [rustup][rustup] to obtain Rust.  Miri comes with a
-`rust-version` file describing the latest supported nightly version of the Rust
-compiler toolchain. Then all you have to do is:
+I recommend that you install [rustup][rustup] to obtain Rust. Then all you have
+to do is:
 
 ```sh
 cargo +nightly build
 ```
 
-with `+nightly` replaced with the appropriate nightly version of Rust.
+This uses the very latest Rust version.  If you experience any problem, refer to
+the `rust-version` file which contains a particular Rust nightly version that
```

Comment:
We could rename the file to `rust-toolchain` and rustup would pick it up automatically. Is there any reason we're not doing that?

Reply:
*lol* we just renamed it *back* from that in the last PR.^^

The reason is that people get really confused when they do `cargo install`, and then use `cargo miri` and nothing works because now they have to manually set the nightly to use to exactly match the one miri uses.

Basically, `rust-toolchain` helps miri developers, but it doesn't help any user that wants to use `cargo miri`. (It helps for really simple examples that work with `cargo run`, but you can just as well use the playground for these.) That seems like the wrong tradeoff.

Reply:
heh ok. makes sense.